importer/importer_utils.py

```diff
@@ -191,6 +191,17 @@ def q_from_first_wikilink(language, text):
         return
 
 
+def get_matching_items_from_dict(value, dict_name):
+    """
+    Return all items in a dict for which the label matches the provided value.
+
+    @param value: the value to match
+    @param dict_name: the dict to look in
+    """
+    return [dict_name[x]["items"]
```

Comment:
is the list comprehension only used in case of a mixture of upper and lower case values?

If so I would collapse it to one list using (but with good variable names):
```python
ll = [dict_name[x]["items"] for x in dict_name if x.lower() == value]
l2 = []
for l in ll:
 l2 += l
return list(set(l2))
```
that way you have only one list (as needed) rather than a nested list.

Reply:
No, there's no check for whether there's a mix... The .lower() is just to make sure if there's an accidental mixture, but there's no rule.

Reply:
Then I would suggest adding on a logic step which returns the 0th element (or an empty list) so that the result is a list rather than a list of lists.

Reply:
Ok.